### PR TITLE
Description of policy for backporting Standard C++ features 

### PR DESCRIPTION
**Pull request by jrhemstad**
closes https://github.com/NVIDIA/libcudacxx/issues/72

### CODE REVIEW

**Comment by wmaxey** [comment]
Verbage looks good to me. The list of backports I think we currently have should be short.

File: docs/standard_api.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ nav_order: 2
...
+## Standard Library Backports
+
+C++ Standard versions include new language features and new library features. 
+As the name implies, language features are new features of the language the require compiler support.
+Library features are simply new additions to the Standard Library that typically do not rely on new language features nor require compiler support and could conceivably be implemented in an older C++ Standard. 
+Typically, library features are only available in the particular C++ Standard version (or newer) in which they were introduced, even if the library features do not depend on any particular language features.
+
+In effort to make library features available to a broader set of users, the NVIDIA C++ Standard Library relaxes this restriction. 
+libcu++ makes a best-effort to provide access to C++ Standard Library features in older C++ Standard versions than they were introduced. 
+For example, the Calendar functionality added to `<chrono>` in C++20 is made available in C++14.
```
Comment (brycelelbach) [inline docs/standard_api.md:18]: Calendar shouldn't be capitalized here.

File: docs/standard_api.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ nav_order: 2
...
+C++ Standard versions include new language features and new library features. 
+As the name implies, language features are new features of the language the require compiler support.
+Library features are simply new additions to the Standard Library that typically do not rely on new language features nor require compiler support and could conceivably be implemented in an older C++ Standard. 
+Typically, library features are only available in the particular C++ Standard version (or newer) in which they were introduced, even if the library features do not depend on any particular language features.
+
+In effort to make library features available to a broader set of users, the NVIDIA C++ Standard Library relaxes this restriction. 
+libcu++ makes a best-effort to provide access to C++ Standard Library features in older C++ Standard versions than they were introduced. 
+For example, the Calendar functionality added to `<chrono>` in C++20 is made available in C++14. 
+
+TODO: List exactly what is backported
```
Comment (brycelelbach) [inline docs/standard_api.md:20]: What are the specific things we've already backported? Let's either start the list, or leave it commented out.

Comment (jrhemstad) [reply docs/standard_api.md]: Added a list of things that are backported.

**Review by miscco — APPROVED** [review]
I have updated the list of those currently backported features that are merged into main.

**Pull request merged**